sdk/src/main/java/com/mapbox/maps/Snapshotter.kt

```diff
@@ -48,6 +52,7 @@ open class Snapshotter {
     snapshotOverlayOptions = overlayOptions
     pixelRatio = context.resources.displayMetrics.density
     coreSnapshotter = MapSnapshotter(options)
+    dispatchTelemetryTurnstileEvent(context, options)
```

Comment:
so new turnstile event is triggered each time when `Snapshotter` is created? I'm not sure it's what we want...
@tobrun shouldn't it be some other telemetry event?

Reply:
+1 - would be nice to get a little more context here

Reply:
This is the same with [V9](https://github.com/mapbox/mapbox-gl-native-android/blob/main/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/snapshotter/MapSnapshotter.java#L416).